pkg/controller/storagecluster/storagecluster_controller_test.go

```diff
@@ -573,3 +574,54 @@ func TestNodeTopologyMapLabelSelector(t *testing.T) {
 	}
 	assert.Equal(t, nodeTopologyMap, sc.Status.NodeTopologies)
 }
+
+// TestStorageClusterOnMultus tests if multus configurations in StorageCluster are successfully applied to CephClusterCR
+func TestStorageClusterOnMultus(t *testing.T) {
+	// When only public network is specified.
+	createStorageClusterWithMultus(t, "public-network", "")
+	// When both public and cluster network are specified.
+	createStorageClusterWithMultus(t, "public-network", "cluster-network")
+	// When only cluster network is specified, this will be an error case .
+	createStorageClusterWithMultus(t, "", "cluster-network")
+	// When public network and cluster network is empty, this will be an error case.
+	createStorageClusterWithMultus(t, "", "")
```

Comment:
Instead of doing multiple function calls, specify this as a case table: https://github.com/openshift/ocs-operator/blob/master/pkg/controller/storagecluster/cephcluster_test.go#L18-L44 You can take the contents of `createStorageClusterWithMultus()` as the body of the `for` loop.